fhirAssembly/src/main/resources/assembly/com.vantiq.fhir.fhirConnection/procedures/com/vantiq/fhir/fhirService_setSource.vail

```diff
@@ -2,11 +2,26 @@ package com.vantiq.fhir
 
 import service com.vantiq.fhir.fhirService
 
-PROCEDURE fhirService.setSource(theSource String)
+PROCEDURE fhirService.setSource(theSource String): String
```

Comment:
Should this be public?  If so, then it probably deserves a "top level" comment like the others.